app/uk/gov/hmrc/individualincomedesstub/domain/Validators.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.individualincomedesstub.domain
+
+import org.joda.time.format.DateTimeFormat
+import org.joda.time.{Interval, LocalDate}
+
+import scala.util.Try
+
+object Validators {
+
+  def valid(requirement: Boolean, message: String) = {
```

Comment:
This implementation of `valid` is identical to `scala.Predef.require`.
So we could reduce code by removing this function and invoking `require` instead.